test/unit/text-field/Input.test.tsx

```diff
@@ -403,3 +405,13 @@ test('#inputElement should return the native input', () => {
   assert.equal(inputElement!.tagName.toLowerCase(), 'input');
   assert.isTrue(inputElement instanceof HTMLInputElement);
 });
+
+test('should call foundation.setValue() when foundation is new', () => {
```

Comment:
This test seems pretty similar to this
https://github.com/material-components/material-components-web-react/blob/c485beacc95aeb3e819a1741888ecaaa8614bb3d/test/unit/text-field/Input.test.tsx#L288

I was referring to the new condition testing that `this.setState({wasUserTriggeredChange: false});` was called.

Reply:
ohh, okay

Reply:
Ya sorry - Also I'm updating this to rc0.11.0_real, which can still go out tomorrow.